config/software/datadog-process-agent.rb

```diff
@@ -0,0 +1,15 @@
+name "datadog-process-agent"
```

Comment:
Nice, you wouldn't have been able to pull from the private repo from circleCI (while building) anyways, so this is the way to go. It'll also keep the build speedy.